src/Resources/scaffolds/6.0/reset-password/src/Form/ResetPasswordFormType.php.tpl

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class ResetPasswordFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('plainPassword', PasswordType::class, [
```

Comment:
Change `PasswordType::class` to `RepeatedType::class` ?
And add `'type' => PasswordType::class,`